src/app.js

```diff
@@ -4,20 +4,32 @@
  */
 
 import { SharedMap } from "fluid-framework";
-import { AzureClient, LOCAL_MODE_TENANT_ID } from "@fluidframework/azure-client";
+import { AzureClient } from "@fluidframework/azure-client";
 import { InsecureTokenProvider } from "@fluidframework/test-client-utils"
 
-// The config is set to run against a local service by default. Run `npx tinylicious` to run locally
```

Comment:
Should we leave in the comment about running `npx tinylicious`?

Reply:
It should be `npx @fluidframework/azure-local-service` either way.

Reply:
I just removed the line since it's a bit confusing. `npm run start` starts both server and client so this line is invalid even in local mode. If we want to define more specifics I think we should just add it to the readme.